### PR TITLE
dislike function

### PR DESCRIPTION
**Pull request by oiz5201618**
add dislike function & change span tag into textarea span to make post flexible

### CODE REVIEW

File: templates/header.html [hunk 1]
```diff
@@ -11,10 +11,18 @@
     <link rel="stylesheet"
           type="text/css"
           href="/static/css/main.css" />
+
+<script>
```
Comment (jserv) [inline templates/header.html:15]: You should split the style changes into another commit.

File: src/main.c [hunk 2]
```diff
@@ -357,6 +360,36 @@ static Response *post(Request *req)
...
+        char sbuff[1024];
+        sprintf(sbuff, "/profile/%d/", post->authorId);
+        bsDel(idStr);
+        return responseNewRedirect(sbuff);
+    }
+
+fail:
+    bsDel(idStr);
+    return responseNewRedirect("/dashboard/");
+
```
Comment (jserv) [inline src/main.c:390]: Don't leave extra blank line.

File: src/models/like.c [hunk 3]
```diff
@@ -44,6 +44,31 @@ Like *likeCreate(sqlite3 *DB, int accountId, int authorId, int postId)
...
 }
 
+Like *dislikeCreate(sqlite3 *DB, int accountId, int authorId, int postId)
+{
+    int rc, t;
+    Like *like = NULL;
+    sqlite3_stmt *statement;
+
+    t  = time(NULL);
+    rc = sqlite3_prepare_v2(
```
Comment (jserv) [inline src/models/like.c:54]: Follow the consistent coding style, please.

File: src/main.c [hunk 4]
```diff
@@ -357,6 +360,36 @@ static Response *post(Request *req)
...
 
+static Response *dislike(Request *req)
+{
+    ROUTE(req, "/dislike/");
+
+    if (!req->account) return NULL;
+
+    int   id = -1;
+    int   idStart = strchr(req->uri + 1, '/') + 1 - req->uri;
+    char *idStr = bsSubstr(req->uri, idStart, -1);
```
Comment (jserv) [inline src/main.c:371]: What is the logic of "dislike" feature? If I "like" a post before, can I "dislike" it later?

File: src/main.c (hunk 2 above)
Comment (oiz5201618) [inline src/main.c:390]: OK,I'll pull a new request

File: src/main.c (hunk 4 above)
Comment (oiz5201618) [inline src/main.c:371]: Yes, you can dislike after you click "like".

File: src/models/like.c (hunk 3 above)
Comment (oiz5201618) [inline src/models/like.c:54]: OK, I'll pull a new request.

File: templates/header.html (hunk 1 above)
Comment (oiz5201618) [inline templates/header.html:15]: OK, I'll fix it now.

File: src/main.c (hunk 4 above)
Comment (jserv) [inline src/main.c:371]: I don't think "like" and "dislike" can be accumulating.
